docs/content/citations.md

```diff
@@ -133,8 +133,11 @@ There are a few ways to reference your book's content, depending on what kind of
 content you'd like to reference. Here is a quick overview of some common options:
 
 * `{ref}` is used to reference section labels that you define or figures with a `name` value
+  * You can reference a section label through ``{ref}`label` `` or ``{ref}`some text <label>` ``
+  * To reference a section label in another document try ``{ref}`path/to/document/label` ``
```

Comment:
I think you can actually just reference the section label directly, no? The `path/to/document/` wouldn't normally be added by Sphinx unless the user defined the label with it. Maybe I'm wrong though?

Reply:
I think you're right but I'll confirm just in case.

Reply:
This has been address in 413e0a9